Source/WebKit/UIProcess/mac/WKFullScreenWindowController.mm

```diff
@@ -299,19 +299,19 @@ - (BOOL)isFullScreen
         || _fullScreenState == InFullScreen;
 }
 
-- (WebCoreFullScreenPlaceholderView *)webViewPlaceholder
+- (RetainPtr<WebCoreFullScreenPlaceholderView>)webViewPlaceholder
```

Comment:
It is fishy to be changing getters to return a RetainPtr, we wouldn't do this in C++ for example due to ref-count churn. Why can't the call sites construct a RetainPtr when they need one?

Reply:
I think fixing the call sites would not get rid of the warning:
https://ews-build.s3-us-west-2.amazonaws.com/macOS-Safer-CPP-Checks-EWS/fdf57ecf-51467/scan-build-output/StaticAnalyzerRegressions/WebKitLegacy/index.html

Now, that may be a bogus warning, since my new PR shows the same kind of warning is not given for a method that does the same thing with same return type.
I'll have a look whether clang-tidy can be fixed to not issue this warning.